src/Components/SampleGallery/SampleGallery.tsx

```diff
@@ -108,7 +108,7 @@ export class SampleGallery extends React.Component<SampleGalleryProps, SampleGal
   public render() {
     return (
       <>
-        <ExpandableList className="gallery-card-radio" singleExpandOnly={true} singleIsCollapsible={true} defaultActiveBlock={0}>
+        <ExpandableList className="gallery-card-radio" singleExpandOnly={true} defaultActiveBlock={0}>
```

Comment:
I think you made this change by accident, possibly through a bad merge.  Please revert.